### ISSUE TITLE
Ethereum explained… to my mom

### ISSUE THREAD

**Issue by ethers** [angelomilan/ethereum-guides#5]
Hi,

Saw your great infographic:
https://d262ilb51hltx0.cloudfront.net/max/800/1*RjOlYNNw_EoDBqPAt6rlog.png

Since it is used in a number of places, there are some spelling mistakes, grammar, and other suggestions that could make it better:
- "network" is more accurate than "net" (hopefully the 4 extra letters don't change the formatting much)
- "message goes through Whatsapp servers."  <- plural with s on servers
- "They are paid to build"    <-  "are" is missing
- "message goes to a network of independent"   <- fixed spelling of independent 
- "Every participating computer" <- fixed spelling of participating
- "Like gas makes your car run."  <- no need for "ing" in running

Great job, and hope that helps.

**Comment by angelomilan** [comment]
hey @ethers (cool nick!) thank you for your suggestions
I'm fixing the infographic right now. I have just 2 questions
"They paid to build"... I wanted to say that servers are not free, they cost money to a company. Therefore, if you use a decentralized network instead of a centralized one, you are supposed to pay.
Do you think it's still wrong... or can you rephrase it in a better way?

"Like gas makes your car run". I admit my ignorance: I was 100% sure it was correct.
Can you explain why I find things like "Your car's engine will stop running." or "How dangerous is it that you leave your car with a running engine,"
What's the difference?
Verbs, nouns... I'm confused a bit :-9

**Comment by ethers** [comment]
Sorry for late response, you're right "They paid to build servers." is good.

"Like gas makes your car running." should be "Like gas makes your car run."
English is confusing.  If you want to use "running", you can do:
"Like gas keeps your car running."
(I think make doesn't usually go with a verb.  For example, you can make a song, but you can't make singing.  [Well, it's tricky because "He makes singing fun" is an ok sentence.]  Hard to explain...)

**Comment by angelomilan** [comment]
Great! I trusted you on the gas thing and fixed already few days ago!
We also have a french version now:
https://medium.com/@angelomilan/ethereum-explained-to-my-mom-infographic-673e32054c1c#.9bsz8enrn
And yeah, I changed the title a bit

**Comment by ethers** [comment]
thanks
